tests/unit/User.test.ts

```diff
@@ -73,16 +71,13 @@ describe('unit', () => {
       beforeEach(() => init())
 
       it('should load a user from serialized wallet', async () => {
-        const loadedUser = await user.load(keystore1)
-        assert.hasAllKeys(loadedUser, ['address', 'serializedWallet', 'pubKey'])
-        assert.isString(loadedUser.address)
-        assert.isString(loadedUser.serializedWallet)
-        assert.isString(loadedUser.pubKey)
+        const loadedUser = await user.loadFrom(TL_WALLET_DATA)
```

Comment:
this should rather test that it is loaded into the library, so something like checking the address would be good.